src/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Todo</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Sharp:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <script
+      src="https://kit.fontawesome.com/0d5e8178bb.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <nav></nav>
+    <div class="to-do-wrapper" id="to-do-wrapper">
+      <div class="h4-refresh">
+        <h3>Today's To do</h3>
+        <i class="fa-solid fa-rotate"></i>
+      </div>
+      <hr />
+      <form action="" class="submit-ToDo" id="submit-ToDo">
+        <input
+          type="text"
+          placeholder="Add to your list"
+          class="Add_To_doList"
+          id="Add_To_doList"
+        />
+        <i class="material-symbols-sharp">keyboard_return</i>
+        <!-- <input type="button" id="addBtn" value="+" /> -->
```

Comment:
## Commented-out code
I suggest you could delete all commented-out codes and I think it could make the code looks clean. Leaving commented-out code in source code is bad practice, as it takes up space, causes confusion, and leads to maintenance issues if it’s not removed. Feel free to leave a comment.